docs/ConvertBMWToJSON.md

```diff
@@ -0,0 +1,45 @@
+# Convert BMW Encoding to JSON Structure
+
+This document describes steps for converting digitalized BMW encoding documents to a JSON file
+that will serve as the foundation for implementing the BMW input method.
+
+BMW encoding documents are in PDF format. These PDFs are composed by digitalized images of orginal
+books. The coversio method is:
+
+1. Split every single page in a PDF into .jpg files
+2. Use OCR library to extract texts from .jpg files
```

Comment:
Do we have a working OCR library for this? Could we provide more detailed instructions?